src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml.cs

```diff
@@ -440,13 +440,13 @@ public A11yElement GetSelectedElement()
         /// <param name="e"></param>
         private void mniShowAncestry_Click(object sender, RoutedEventArgs e)
         {
+            SetFocusOnHierarchyTree();
             Configuration.ShowAncestry = this.mniShowAncestry.IsChecked;
             if (this.SelectedElement != null)
             {
                 var dic = new Dictionary<string, string>();
                 this.HierarchyActions.RefreshHierarchy(false);
             }
-            SetFocusOnHierarchyTree();
```

Comment:
This was actually a NOP, since the focus had been shifted on menu load. The focus shift needs to occur before refreshing the hierarchy, hence setting the focus earlier in this method.